src/cfe.c

```diff
@@ -682,12 +597,13 @@ void et_from_rainfall(double *timestep_rainfall_input_m, struct evapotranspirati
         }
 
         else{
-
-            et_struct->potential_et_m_per_timestep -= *timestep_rainfall_input_m;
+            // LKC: This was incorrectly set to potential instead of actual
+	    et_struct->actual_et_m_per_timestep = *timestep_rainfall_input_m;          

```

Comment:
Just be sure is this intended to be a copy of `timestep_rainfall_input_m`?  The previously incorrect `potential` variable looked like an accumulation using `-=`, where as now there is no cumulative effect, just a straight assignment using `=`.

Reply:
Yes, it was intended to copy timestep_rainfall_input_m. The previous statement with -= was moved outside of the loop to line 606 since it is valid for both conditions.